Sources/DesignSystem/Views/BraveTextFieldStyle.swift

```diff
@@ -110,3 +110,49 @@ private struct BaseBraveTextFieldStyleModifier: ViewModifier {
       .clipShape(borderShape)
   }
 }
+
+public struct BaseBraveTextEditorStyleModifier: ViewModifier {
```

Comment:
This doesnt make sense to have, if you want a TextEditor w/ placeholder component make a separate `View` not a modifier. Also wouldn't go in this file and wouldnt be named a TextEditorStyle

Reply:
I was hoping we could use `TextEditorStyle` but I don't know if this is just not fully implemented. I can't seem to get this to work. But ok, I moved to using just a View called `BraveTextEditor`. I seen other places we use the text editor and I think this should be reusable.